comp/core/tagger/taggerimpl/collectors/workloadmeta_test.go

```diff
@@ -2126,7 +2126,7 @@ func TestHandleContainer(t *testing.T) {
 					Name: containerName,
 				},
 				Resources: workloadmeta.ContainerResources{
-					GPUTypeList: []string{"nvidia"},
+					GPUVendorList: []string{"nvidia"},
```

Comment:
You'll also need to change the "kube_gpu_type" included below in `LowCardTags` from `expected`.

Reply:
yes indeed